cubeviz/data_factories/configurations/jwst-fits.yaml

```diff
@@ -0,0 +1,35 @@
+# JWST FITS cube
+name: 'jwst-fits-cube'
+type: 'JWST-FITS-CUBE'
+
+# Highest priority data configuration file, that matches, is selected.
+priority: 1200
+
+# All must match and they must have SCI, ERR, DQ extensions
+match:
```

Comment:
Eventually we will need to figure out a way to handle other data formats such as `ASDF` and possibly `HDF5`.

Reply:
What is ASDF?

Reply:
bhahahahahahhahaa